Flagsmith.FlagsmithClient/FlagsmithConfiguration.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+namespace Flagsmith
+{
+    public class FlagsmithConfiguration
+    {
+        public FlagsmithConfiguration()
+        {
+            ApiUrl = "https://api.flagsmith.com/api/v1/";
+            EnvironmentKey = string.Empty;
+            UseLegacyIdentities = true;
```

Comment:
What's this?

Reply:
for `UseLegacyIdentities`, I missed to discuss that property, as in previous dotnet implementation this property is used to call either depreciated or new Identities API url in `GetIdentitiesUrl()`. should i remove that ?

Reply:
Yes, this client should only use `/api/v1/identities/`.